sql/src/main/java/io/crate/analyze/OrderBy.java

```diff
@@ -160,7 +160,11 @@ public OrderBy copy() {
         return newOrderBy;
     }
 
-    public OrderBy copyAndReplace(Function<? super Symbol, Symbol> replaceFunction) {
+    /**
+     * Create a new "copy" of the OrderBy by replacing the orderBy symbols
+     * using the replaceFunction without affecting the original symbols.
```

Comment:
maybe note that it's up to the given function to do a copy or not (and maybe that the reverseFlags and nullsFirst arrays are not copied)